WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -505,6 +506,7 @@ public void run() {
     private void initializePostObject() {
         if (mPost != null) {
             mOriginalPost = mPost.clone();
+            mOriginalPostHadLocalChangesOnOpen = mOriginalPost.isLocallyChanged();
```

Comment:
What about naming `mOriginalPostHadLocalChangesOnOpen` something like `mOriginalPostWasLocallyChanged` since its only assignment is `mOriginalPost.isLocallyChanged()`?

Reply:
Thanks for the suggestion @theck13 ! I think the current name, while a bit lengthier than the suggested one, better describes the specific situation where it's going to be used - that is, the value it's carrying is only meaningful `OnOpen`. We could either leave it as is or maybe rename it to `mOriginalPostWasLocallyChangedOnOpen` if you want? wdyt?

Reply:
It's not a big deal so we can leave it as is.